### PR TITLE
[Remove Stream/Optional usage] LPS-171374  headless PART|1

### PR DESCRIPTION
**Pull request by JanisZhang**
Remove usages of Stream on Method Return Value for module: `headless` PART 1.

https://issues.liferay.com/browse/LPS-171374

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:help

**Comment by JanisZhang** [comment]
ci:test:headless

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/KnowledgeBaseArticleDTOConverter.java [hunk 1]
```diff
@@ -99,13 +99,15 @@ public KnowledgeBaseArticle toDTO(DTOConverterContext dtoConverterContext)
...
-				numberOfAttachments = Optional.ofNullable(
-					kbArticle.getAttachmentsFileEntries()
-				).map(
-					List::size
-				).orElse(
-					0
-				);
+
+				numberOfAttachments = 0;
+				List<FileEntry> fileEntries =
```
Comment (yuhai) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/KnowledgeBaseArticleDTOConverter.java:104]: @JanisZhang, the code looks good to me. could you please keep code format consistent with https://github.com/liferay-core-infra/liferay-portal/pull/2860/commits/d3fcb1036929f716e7bebd58ddc694c76e3ca763. After you push the changes, you may only execute ci:test:sf

Comment (JanisZhang) [reply modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/KnowledgeBaseArticleDTOConverter.java:104]: Hi @yuhai, force-pushed. Thanks!

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:sf

**Review by yuhai — APPROVED** [review]

**Comment by yuhai** [comment]
@tinatian, @dantewang, the fix looks good to me. Thanks.

**Comment by yuhai** [comment]
@tinatian, @dantewang, the fix looks good to me. Thanks.

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:forward

**Comment by dantewang** [comment]
@zhangyan0701 Could you take a look at the latest results again?

**Comment by zhangyan0701** [comment]
The failures are unrelated.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/128385

**Pull request closed without merging**